specs/splat_attention_and_matmul_spec.mlir

```diff
@@ -540,7 +540,7 @@ module attributes { transform.with_named_sequence } {
         workgroup_size = [64, 2, 1] subgroup_size = 64,
         {mma_schedule = #iree_gpu.mma_schedule<
            intrinsic = #iree_gpu.mma_layout<MFMA_F16_16x16x16_F32>,
-           subgroup_m_count = 2, subgroup_n_count = 1>
+           subgroup_m_count = 4, subgroup_n_count = 1>
```

Comment:
Have you tried compiling with this script? The workgroup size does not match the subgroup counts. I would expect `64, 4, 1`